### PR TITLE
Makman2/linters5

### PR DESCRIPTION
**Pull request by Makman2**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
ack d8815c3

File: bears/html/HTMLLintBear.py [hunk 1]
```diff
@@ -1,33 +1,26 @@
...
-        "Warning": RESULT_SEVERITY.NORMAL,
-        "Error": RESULT_SEVERITY.MAJOR
-    }
+@linter(executable=sys.executable,
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):(?P<column>\d+): '
+                     r'(?P<severity>Error|Warning|Info): (?P<message>.+)')
+class HTMLLintBear:
+    """
+    Checks the code with ``html_lint.py`` on each file separately.
```
Comment (Adrianzatreanu) [inline bears/html/HTMLLintBear.py]: are we not going with imperative? ``Check``

Comment (Adrianzatreanu) [inline bears/html/HTMLLintBear.py]: ah this is changed in the other commit.

File: bears/html/HTMLLintBear.py [hunk 2]
```diff
@@ -1,33 +1,26 @@
...
-    )
-    severity_map = {
-        "Info": RESULT_SEVERITY.INFO,
-        "Warning": RESULT_SEVERITY.NORMAL,
-        "Error": RESULT_SEVERITY.MAJOR
-    }
+@linter(executable=sys.executable,
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):(?P<column>\d+): '
+                     r'(?P<severity>Error|Warning|Info): (?P<message>.+)')
```
Comment (Adrianzatreanu) [inline bears/html/HTMLLintBear.py]: i'm still asking if this works for sure, with spaces instead of `\s` here..

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/356/commits/539b3f0b217a9b5f912f08b6091d475c9a249b21

File: bears/html/HTMLLintBear.py (hunk 2 above)
Comment (sils1297) [inline bears/html/HTMLLintBear.py]: well it has a space always anyway, tests verify that

Comment (Makman2) [inline bears/html/HTMLLintBear.py]: Yes it works, tested locally

File: bears/go/GoVetBear.py [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
-from coalib.bears.LocalBear import LocalBear
 
+@linter(executable='go',
+        use_stdout=False,
+        use_stderr=True,
+        output_format='regex',
+        output_regex=r'.+:(?P<line>\d+): (?P<message>.*)')
+class GoVetBear:
+    """
+    Checks the code using ``go vet``.
```
Comment (Adrianzatreanu) [inline bears/go/GoVetBear.py]: needs another commit for imperative.

**Comment by Adrianzatreanu** [comment]
ack f5900c1b

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**